control/control.go

```diff
@@ -233,12 +233,12 @@ func (c *Controller) Solve(ctx context.Context, req *controlapi.SolveRequest) (*
 	if err != nil {
 		return nil, err
 	}
-	if req.Exporter != "" {
-		exp, err := w.Exporter(req.Exporter, c.opt.SessionManager)
+	if req.Exporters[0] != "" {
+		exp, err := w.Exporter(req.Exporters[0], c.opt.SessionManager)
```

Comment:
Afaics this exports to first item of the array, not to all items.

Reply:
Done. For the moment I commented some parts in code, I will reactive them.